### PR TITLE
Getting Started with PHP

### LINKED ISSUE #1378: Getting Started with PHP
### Brief Summary: 
PHP is a server scripting language and a powerful tool for making dynamic and interactive web pages. It is  widely-used, free, and efficient language.It is also easy to learn and understand hence making it one of the best server language for beginners. PHP is free and easy to download. In this tutorial we will look at the most important basic parts of PHP step by step.

### Key Takeaways: 
After reading this article the reader will be able to:
1. set up an environment for running PHP i.e downloading xampp which will come along with PHP
2. understand PHP syntax
3. write a program eg Helloworld using PHP
4. write PHP statements
5. output a PHP program

### References: 
N/A

### PR DESCRIPTION
**Pull request by Erastuskanyatta**
### Article checklist
**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you placed your article through a 3rd party plagiarism checker? We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [ ] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
this fixes #1378

### CODE REVIEW

**Comment by hectorkambow** [comment]
plagiarism check done

File: articles/Introduction-to-PHP/index.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+<?php
+$age = 15;
+
+if ($age < 18) {
+ echo "a kid!";
+}
+?>
+```
+Running this code gives `a kid!` as the output.
+### 2. ````if...else```` statement
```
Comment (geoffrey45) [inline articles/Introduction-to-PHP/index.md:114]: Since this is a heading please write this code block as plain text.

**Review by geoffrey45 — CHANGES_REQUESTED** [review]
@Erastuskanyatta Nice article. Just some minor changes. Please work on the suggestions and ping me when you are through.

**Comment by Erastuskanyatta** [comment]
@geoffrey45 I have completed the changes you requested. Please continue with the review. Thank you

**Review by geoffrey45 — APPROVED** [review]

**Pull request merged**